src/test/scala/tech/cryptonomic/conseil/tezos/PlatformDiscoveryOperationsTest.scala

```diff
@@ -163,4 +171,83 @@ class PlatformDiscoveryOperationsTest
     }
   }
 
+  "listAttributeValues" should {
+
+    "return list of values of kind attribute of Fees without filter" in {
+      val avgFee = AverageFees(1, 3, 5, Timestamp.valueOf(LocalDateTime.of(2018, 11, 22, 12, 30)), "example1")
+
+      Await.result(dbHandler.run(TezosDatabaseOperations.writeFees(List(avgFee))), Duration.Inf)
+      dbHandler.run(
+        DBIO.sequence(
+          PlatformDiscoveryOperations.verifyAttributesAndGetQueries("fees", "kind", None)
+        )
+      ).futureValue.flatten shouldBe List("example1")
+    }
+
+    "return throw an exeption when asked for medium attribute of Fees without filter - numeric attributes should not be displayed" in {
```

Comment:
typo: "return throw an exception when..." -> "throw an exception when..."

personally I'd say that it "returns a failed future when..."

Reply:
changed